README.md

```diff
@@ -1,14 +1,14 @@
-NixStats Agent
+Agent360
 ==============
 
-NixStats.com is a web service of monitoring and displaying statistics of
+Agent360 is a web service of monitoring and displaying statistics of
```

Comment:
This actually is not true. The agent is just an agent. The 360 platform is a service.